framework/utils/randomUtils.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+ This file contains the random number generating methods used in the framework.
+ created on 07/15/2017
+ @author: talbpaul
+'''
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+
+import numpy as np
+from collections import deque
+
+from utils.utils import find_distribution1D
+
+# in general, we will use Crow for now, but let's make it easy to switch just in case it is helpfull eventually.
+stochasticEnv = 'crow'
+#stochasticEnv = 'numpy'
```

Comment:
I did try this personally, and it worked as you ask; however, I did not add tests for the numpy side, as there's no current plan to use numpy RNG over crow RNG currently.  The numpy side was developed before I understood the scope of our RNG usage in RAVEN.